pages/knowledge/colours-and-contrast/graphical-objects/README.md

```diff
@@ -0,0 +1,50 @@
+
+## Example: line chart
+
+The following picture shows a simple line chart with overall insufficient contrast. 
+
+![Line chart with one blue trend line and overall low contrast](_media/line-chart-1-low-contrast.png)
+
+The contrast ratio of the main X and Y axis is `2.8:1`, the one of blue trend line is `2.3:1`. The text labels along the X and Y axis have a contrast ratio of `3.9:1` which can also be considered too low because [text has to meet a higher contrast requirement](/knowledge/colours-and-contrast/text) of at least `4.5:1`.  
+All of these values are somewhat close to their formal requirements but it’s important to keep in mind that those requirements only set *minimal* standards. 
+
+Let’s increase the contrast ratio of the aforementioned elements.
+
+![The same line chart with increased contrast for the blue trend line and the x and y axis](_media/line-chart-2-partial-contrast.png)
+
+The question now is, what should happen with the grid lines in the background. Their contrast ratio is far too low, yet increasing it might result in a visually overloaded chart.
+
+This brings us back to the notion of what is “required to understand the meaning or content” of this chart. 
+
+If the purpose of the chart is to just show a general trend, the grid lines might not be required. In that case, why not just remove them altogether? (If that feels like a loss, this might not be the right solution.)
+
+However, if it is in fact important that people can comfortably read the exact values of each measuring point, it might be a good start to mark those points.
+
+![The same line chart with added dots along the trend line to mark measuring points](_media/line-chart-3-partial-contrast-dots.png)
+
+Horizontal grid lines might make it easer to gauge the value of the measuring point without overloading the overall appearance.
+
+![The same line chart with horizontal grid lines, indicating the steps on the y axis](_media/line-chart-4-full-contrast-dots.png)
+
+Or why not add a label to each measuring point, indicating its value?
+
+![The same line chart with labelled measuring points, as well as vertical grid lines, indicating the steps on the x axis.](_media/line-chart-5-full-contrast-dots-labels.png)
+
+It’s no coincidence that many of these considerations and adjustments go far beyond just colour contrast. Designing accessible user interfaces requires a holistic approach. This means:
+
+- There is rarely a single variable that can be independently tweaked. Accessibility should be built into the foundations of the design.
+- There is rarely one generic solution that “just works”. The specific context and content have to be taken into account.
+- There is no strict separation between “special-purpose accessibility” and “general-purpose usability”. In practice they both influence each other.
```

Comment:
We should also point out, that strictly speaking, WCAG defines alternative ways of accessing an information as enough to also meet a criterion. For example, delivering a downloadable CSV- or Excel-file with the data that underlies a chart could be seen as "alternative way". This is a dangerous one though, as not every user might be able to download and interpret such a file...

Reply:
**Question:** Is this point still part of “Colours and Contrast”?

Reply:
Too specific.